packages/x-date-pickers/src/PickersDay/PickersDay.tsx

```diff
@@ -495,11 +495,10 @@ PickersDayRaw.propTypes = {
 } as any;
 
 /**
- *
  * Demos:
  *
- * - [Date Picker](https://mui.com/x/react-date-pickers/date-picker/)
- *
+ * - [DatePicker](https://mui.com/x/react-date-pickers/date-picker/)
```

Comment:
WDYT about removing the `DatePicker` page as it technically doesn't show anything more than the specific `DateCalendar` page? 🤔 
```suggestion
```